wand/image.py

```diff
@@ -2190,7 +2194,12 @@ def composite(self, image, left, top):
             raise TypeError('left must be an integer, not ' + repr(left))
         elif not isinstance(top, numbers.Integral):
             raise TypeError('top must be an integer, not ' + repr(left))
-        op = COMPOSITE_OPERATORS.index('over')
+        try:
+            op = COMPOSITE_OPERATORS.index(operator)
+        except IndexError:
+            raise IndexError(repr(operator) + ' is an invalid composite '
+                             'operator type; see wand.image.COMPOSITE_'
+                             'OPERATORS dictionary')
```

Comment:
The introduced new behavior also need to be unit-tested.  Could you write some tests for it?